src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -143,9 +144,19 @@
      * will cause VariantsToTable to write out NA values for missing fields instead of throwing an error.
      * Note that this flag only applies to standard columns (CHROM, ID, QUAL) and the INFO field and it does not apply to the genotype field.
      */
-    @Advanced
-    @Argument(fullName="allowMissingData", shortName="AMD", doc="If provided, we will not require every record to contain every field", optional=true)
+    @Hidden
+    @Deprecated
+    @Argument(fullName="allowMissingData", shortName="AMD", doc="This argument is no longer used, refer to the documentation for --errorIfMissingData", optional=true)
```

Comment:
@vdauwera I don't think you'll get a nice message the way it is right now.  I think it will just silently continue.  You'd have to decide to look at the documentation, while enabling `--showHidden` in order to see the message.  ( I don't think barclay understands automatic warning on deprecated arguments...  )  If we want to show a message for a while as a transitionary step we'd have to add one manually.  I'd vote for just killing it though.